coral-component-colorpicker/src/scripts/ColorProperties.js

```diff
@@ -205,9 +206,21 @@ class ColorProperties extends BaseComponent(HTMLElement) {
 
   /** @private */
   _onColorInputChange(event) {
+    var inputColorText = this._colorInput.value;
+    var color = new TinyColor(inputColorText);
+    if (!color.isValid) {
+      return;
+    }
+    var cursorLoc = event.target.selectionStart;
     event.stopImmediatePropagation();
     this.color = this._colorInput.value;
     this.trigger('change');
+    // trigger picker change event & send input color as event details to set it in picker input
+    this.trigger('change', inputColorText);
```

Comment:
why are we sending change event twice, you can send only 219.

Reply:
thanks for noticing. Incorporated.